README.md

```diff
@@ -1 +1,51 @@
 # project-liberation
+
+## Development setup
+
+### Preparing your environment for development
+
+1.  Install pipenv on your machine. Do not create any virtualenv, just install globally:
+       
+    Manjaro linux:  
+    ``` bash
+     sudo pacman -Syu python-pipenv
+    ```
+    Ubuntu: 
+    ``` bash
+     pip install pipenv
+    ```
+
+2.  Create new virtual environment for project with all development packages. 
+    ``` bash
+     pipenv install --dev
+    ```
+        
+    Alternatively, this can be also done using Pycharm IDE.  <br />
+    - Click "Add Interpreter" (right down corner) <br />
+    - Select Pipenv Environment <br />
+    - Choose python version, make sure that checkbox "Install packages from Pipfile" is selected <br />
+    - Confirm with OK <br />
+    New virtual environment will be created. Pycharm terminals will use this virtualenv, so there is no need to proceed step 3 any more. Skip to step 4
```

Comment:
Please delete this because this is unnecessary to simple development instruction.

Reply:
I disagree, virtualenv can be created in two ways and IDE is useful tool for this. I will keep this, if you think it should be deleted anyway please remove it before merge.

Reply:
Done